app.js

```diff
@@ -23,63 +34,144 @@ var stuff = [
   new RandomPicture('scissors', 'img/scissors.jpg'),
   new RandomPicture('shark', 'img/shark.jpg'),
   new RandomPicture('sweep', 'img/sweep.png'),
-  new RandomPicture ('tauntaun', 'img/tauntaun.jpg'),
-  new RandomPicture ('unicorn', 'img/unicorn.jpg'),
+  new RandomPicture('tauntaun', 'img/tauntaun.jpg'),
+  new RandomPicture('unicorn', 'img/unicorn.jpg'),
   new RandomPicture('usb', 'img/usb.gif'),
   new RandomPicture('waterCan', 'img/waterCan.jpg'),
   new RandomPicture('wineGlass', 'img/wineGlass.jpg')
 ];
 
-// for(var i = 0; i < stuff.length; i++){
-//   // console.log(stuff[i].name);
-//   // console.log(stuff[i].src);
-// }
-
 function findRandom(){
-  var tempArray = [];
-  while(tempArray.length < 3){
-    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
-    if(tempArray.indexOf(rand) === -1){
-      tempArray.push(rand);
-    }
-    // console.log(tempArray);
-  }
-  randomNumArray = tempArray;
+  // var tempArray = [];
+  // while(tempArray.length < 3){
+    // var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+  return(Math.floor(Math.random() * stuff.length));
```

Comment:
Return isn't a function, but it looks like you might be treating it as one. It would be cleaner to write:
```javascript
function findRandom(){
  return Math.floor(Math.random() * stuff.length);
}
```